customer-chat/app.js

```diff
@@ -0,0 +1,196 @@
+/**
+ * Copyright 2017-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ * Starter Project for Messenger Platform Quick Start Tutorial
+ *
+ * Use this project as the starting point for following the
+ * Messenger Platform quick start tutorial.
+ *
+ * https://developers.facebook.com/docs/messenger-platform/getting-started/quick-start/
+ *
+ */
+
+'use strict';
+
+// Imports dependencies and set up http server
+const
+    request = require('request'),
+    express = require('express'),
+    body_parser = require('body-parser'),
+    app = express().use(body_parser.json()); // creates express http server
+app.use(express.static('public'));
+
+// TODO: Add values here
+const access_token = '';
```

Comment:
group your consts with the others at the top before the app.use

Reply:
@amuramoto OK, but again it's like this in the docs and other tutorials, so might be worth checking.

Reply:
Maybe 🤔